pupil_src/shared_modules/calibration_routines/calibrate.py

```diff
@@ -17,6 +17,9 @@
 
 logger = logging.getLogger(__name__)
 
+not_enough_data_error_msg = (
```

Comment:
This variable is only used inside of `match_data`. Is there a reason for it to be defined globally rather than inside of `match_data`?

Reply:
Only historical ones. I have moved it as you proposed.